modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,178 @@
+[source,terminal]
+----
+$ tkn pipelinerun delete -all --keep-since 5
+----
+
+* With this update, when you use the `tkn pipelinerun delete --all` command, you can use the new `--label <string>` option to filter the pipeline runs by label. Optionally, you can use the `--label` option with `=` and `==` as equality operators, or `!=` as an inequality operator. For example, the `tkn pipelinerun delete --all --label asdf` and  `tkn pipelinerun delete --all --label==asdf` commands both delete all the pipeline runs that have the `asdf` label. (link:https://github.com/tektoncd/cli/pull/1402[#1402])
+
+* With this update, you can fetch the version of installed Tekton components from the config map or, if the config map is not present, from the deployment controller. (link:https://github.com/tektoncd/cli/pull/1393[#1393])
+
+[id="new-features-tekton-triggers-0-16-0-release-1-6_{context}"]
+=== Tekton Triggers v0.16.0
+
+* With this update, triggers support the `feature-flags` and `config-defaults` config map to configure feature flags and to set default values respectively. (link:https://github.com/tektoncd/triggers/pull/1182[#1182], link:https://github.com/tektoncd/triggers/pull/1110[#1110])
+
+* This update adds a new metric, `eventlistener_event_count`, that you can use to count events received by the `EventListener` resource. (link:https://github.com/tektoncd/triggers/pull/1160[#1160])
+
+[id="new-features-tekton-triggers-0-15-0-release-1-6_{context}"]
+=== Tekton Triggers v0.15.0
+
+* This update adds `v1beta1` Go API types. With this update, triggers now support the `v1beta1` API version.
++
+With the current release, the `v1alpha1` features are deprecated and are candidates for removal after nine months. Begin using `v1beta1` features instead. (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
+
+[id="new-features-pipelines-operator-1-6_{context}"]
+=== {pipelines-title} Operator
+
+* The current release enables you to configure node selection by specifying the `nodeSelector` and `toleration` values in the `TektonConfig` custom resource (CR). The Operator adds these values to all deployments it creates.
++
+To configure node selection for the Operator's controller and webhook deployment, you edit the `config.nodeSelector` and `config.toleration` fields in the specification for the `Subscription` CR after installing the Operator.
++
+To deploy on an infra node, you update the `TektonConfig` and `Subscription` CRs with the infra label as `nodeSelector` and `toleration` so that these labels are applied to the infra node. (link:https://issues.redhat.com/browse/SRVKP-1544[SRVKP-1544])
+
+[id="deprecated-features-1-6_{context}"]
+== Deprecated features
+
+[id="deprecated-cli-0-21-0-release-1-6_{context}"]
+=== CLI 0.21.0 release
+
+* In CLI 0.21.0, support for all `v1alpha1` resources for `clustertask`, `task`, `taskrun`, `pipeline`, and `pipelinerun` commands are deprecated. These resources will be removed after two releases.
```

Comment:
`These resources will be removed after two releases.`  - does this refer to upstream releases or product releases? In previous releases we only stated something like `... are now deprecated and will be removed in a future release.`

Reply:
Updated throughout. Thanks.